templates/design_system/authorize_v2.html

```diff
@@ -27,28 +27,33 @@
 		<h1 class="ds-u-font-size--h2">{{ application.name }} has asked for some of your data.</h1>
 
 		<ul>
-			<li>Information about your doctor/hosiptal visits</li>
+			<li>Information about your doctor/hospital visits</li>
 			<li>Information about the prescription medications you take</li>
-			<li>Personal Information like your name, address, date of birth, race, and gender</li>
+			{% if application.require_demographic_scopes != False %}
```

Comment:
Stylistic choice but I would avoid using `!=` in truth testing to make this simpler.

Reply:
@njdister Looks like this template got copied over from the P2 scopes PR.  The `application.require_demographic_scopes` field does not exist yet.  I think that would be a good style change for the P2 scopes PR, where that boolean field is available. Perhaps that code should be removed here, since it doesn't apply until P2?

Reply:
👍